ivy_tests/test_ivy/test_frontends/test_numpy/test_indexing_routines/test_indexing_like_operations.py

```diff
@@ -139,3 +139,43 @@ def test_numpy_put_along_axis(
         axis=axis,
         values=values,
     )
+
+
+@handle_frontend_test(
+    fn_tree="numpy.compress",
+    dtype_arr_ax=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_num_dims=1,
+        max_num_dims=1,
```

Comment:
Setting `min_num_dims=max_num_dims=1` means we are only testing for 1D arrays, while the function must work for any array shape.

Reply:
oh that's an oversight